jsplot/src/main/java/tech/tablesaw/plotly/components/Figure.java

```diff
@@ -110,6 +127,20 @@ private String plotFunction(String divName) {
         }
 
         builder.append(");");
+        
+        return builder.toString();
+    }
+
+    private String eventHandlerFunction(String targetName, String divName) {
+        StringBuilder builder = new StringBuilder();
+        
+        if (eventHandlers != null ) {
+            builder.append(System.lineSeparator());
+            for (int i=0; i < eventHandlers.length; i++) {
+                builder.append( eventHandlers[i].asJavascript(targetName, divName) );
```

Comment:
no need for space before and after parentheses

Reply:
check